backends-velox/src/main/scala/org/apache/gluten/config/VeloxConfig.scala

```diff
@@ -306,6 +317,21 @@ object VeloxConfig {
       .intConf
       .createOptional
 
+  val COLUMNAR_VELOX_RESIZE_BATCHES_SHUFFLE_INPUT_OUTPUT_MIN_SIZE =
```

Comment:
`s/SHUFFLE_INPUT_OUTPUT/SHUFFLE_OUTPUT` ?

Reply:
This is used both for the BatchResizing before shuffle input and after shuffle output, so that we can reduce some config. Usually there is need to do too much customized min size config for these two scenario. What do you think?